cli/azd/pkg/input/muted_console.go

```diff
@@ -0,0 +1,82 @@
+package input
```

Comment:
FWIW - Muted Console is no longer used in the codebase.  It existed to solve the problem of having us re-use much of the bicep deployment logic for updating container apps, but we no longer do this.

I think I have a slight preference towards just deleting this unused type.

I think it's existence is a bit of of a code smell and while it was the right tactical move at the time, we no longer need it and so I'd like to just make it harder to folks to take a dependency on it.